examples/magento-graphcms/pages/checkout/added.tsx

```diff
@@ -24,6 +25,11 @@ type GetPageStaticProps = GetStaticProps<LayoutOverlayProps, Props>
 
 function CheckoutAdded() {
   const [addedItem, crossSellItems] = useCrosssellItems()
+  const a11yFocusRef = useRef<HTMLHeadingElement | null>(null)
+
+  useEffect(() => {
+    a11yFocusRef.current?.focus()
+  })
```

Comment:
Not passing a dependency array will cause this to happen every render. Should this not happen only on "mount" (by adding an empty dependency array)?